cpp/src/groupby/groupby.cu

```diff
@@ -99,6 +99,8 @@ namespace {
 struct empty_column_constructor {
   column_view values;
   aggregation const& agg;
+  rmm::cuda_stream_view stream;
+  rmm::device_async_resource_ref mr;
```

Comment:
Why would creating an empty column require a stream or mr?

Reply:
The logic here is, we need to construct an "empty output" for the UDF, not just "empty column". Such "empty output` can be anything, such as a column of all nulls, depending on the definition of such UDF. So we need to pass in a memory resource.